### ISSUE TITLE
BUG: Minor HTML formatting error in Styler.render(): rowspan attribute should use quotes

### ISSUE THREAD

**Issue by fwt** [pandas-dev/pandas#38234]
- [x] I have checked that this issue has not already been reported.

- [x] I have confirmed this bug exists on the latest version of pandas.

- [x] (optional) I have confirmed this bug exists on the master branch of pandas.

---

**Note**: Please read [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) detailing how to provide the necessary information for us to reproduce your bug.

#### Code Sample, a copy-pastable example

```python
df = pd.DataFrame(data=[[1, 2]], index=[["l0", "l0"], ["l1a", "l1b"]])
s = Styler(df, uuid="_", cell_ids=False)
assert '<th id="T__level0_row0" class="row_heading level0 row0" rowspan="2">l0</th>' in s.render()
```

#### Problem description

This issue is very similar to #35706. The ``colspan`` attribute has been fixed in #36223 but the ``rowspan`` attribute hasn't been fixed yet. Fixing this would improve HTML correctness and make the ``Styler.render()`` behavior more consistent.

Compare [line 330](https://github.com/attack68/pandas/blob/42f600e2acc0cd905c078ab77b473f0537699cc5/pandas/io/formats/style.py#L330) (colspan attribute) and [line 376](https://github.com/attack68/pandas/blob/42f600e2acc0cd905c078ab77b473f0537699cc5/pandas/io/formats/style.py#L376) (rowspan attribute).

#### Expected Output

The ``rowspan`` attribute uses quotes and the assert statement doesn't fail.

#### Output of ``pd.show_versions()``

<details>

INSTALLED VERSIONS
------------------
commit           : db08276bc116c438d3fdee492026f8223584c477
python           : 3.8.5.final.0
python-bits      : 64
OS               : Linux
OS-release       : 4.19.76-linuxkit
Version          : #1 SMP Tue May 26 11:42:35 UTC 2020
machine          : x86_64
processor        : 
byteorder        : little
LC_ALL           : None
LANG             : C.UTF-8
LOCALE           : en_US.UTF-8

pandas           : 1.1.3
numpy            : 1.19.2
pytz             : 2020.1
dateutil         : 2.8.1
pip              : 20.1.1
setuptools       : 50.3.0
Cython           : None
pytest           : None
hypothesis       : None
sphinx           : None
blosc            : None
feather          : None
xlsxwriter       : None
lxml.etree       : None
html5lib         : None
pymysql          : None
psycopg2         : None
jinja2           : 2.11.2
IPython          : 7.18.1
pandas_datareader: None
bs4              : None
bottleneck       : None
fsspec           : None
fastparquet      : None
gcsfs            : None
matplotlib       : 3.3.2
numexpr          : None
odfpy            : None
openpyxl         : None
pandas_gbq       : None
pyarrow          : None
pytables         : None
pyxlsb           : None
s3fs             : None
scipy            : 1.5.2
sqlalchemy       : None
tables           : None
tabulate         : None
xarray           : None
xlrd             : None
xlwt             : None
numba            : None

</details>

**Comment by JoseNavy** [comment]
I was able to reproduce this bug in master, so I will try to fix it!

**Comment by simonjayhawkins** [comment]
closed in #38533
